conf/st2.dev.conf

```diff
@@ -5,7 +5,6 @@
 host = 0.0.0.0
 port = 9101
 logging = st2api/conf/logging.conf
-serve_webui_files = True
 # allow_origin is required for handling CORS in st2 web UI.
 # allow_origin = http://myhost1.example.com:3000,http://myhost2.example.com:3000
```

Comment:
Yeah, I agree with @DoriftoShoes on this one.

As an extra precaution, we could print a "don't use this in prod" warning during st2api startup if `allow_origin = *`.